stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/selectedmessage/SelectedMessageMenu.kt

```diff
@@ -39,10 +31,8 @@ import io.getstream.chat.android.compose.ui.theme.ChatTheme
  * @param overlayColor The color applied to the overlay.
  * @param reactionTypes The available reactions within the menu.
  * @param onDismiss Handler called when the menu is dismissed.
- * @param headerContent Leading vertical Composable that allows the user to customize the content shown in [SelectedMessageOptions].
- * By default [ReactionOptions].
- * @param bodyContent Trailing vertical Composable that allows the user to customize the content shown in [SelectedMessageOptions].
- * By Default [MessageOptions].
+ * @param headerContent The content shown at the top of the [SelectedMessageMenu] dialog. By default [ReactionOptions].
+ * @param bodyContent The content shown in the [SelectedMessageMenu] dialog. By Default [MessageOptions].
  */
 @Composable
 public fun SelectedMessageMenu(
```

Comment:
As marin pointed out - I'd follow the new practice here - have the `DefaultSelectedMessageHeaderContent` and `DefaultSelectedMessageBodyContent` composables as internal and then use those for the default slots here.